CHANGELOG.md

```
@@ -4,6 +4,22 @@
     Please refer to https://github.com/truefoundry/KubeElasti/blob/main/CONTRIBUTING.md#Changelog and follow the guidelines before adding a new entry.
 -->
 
+All the unreleased changes are listed under `Unreleased` section. Add your changes here, they will be moved to the next release. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)